src/icons/Status.js

```diff
@@ -74,23 +74,27 @@ export function Error({ className }) {
     return (
         <svg
             xmlns="http://www.w3.org/2000/svg"
-            width="48"
-            height="48"
             viewBox="0 0 48 48"
             className={className}
         >
             <path d="M24 4C12.96 4 4 12.95 4 24s8.96 20 20 20 20-8.95 20-20S35.04 4 24 4zm2 30h-4v-4h4v4zm0-8h-4V14h4v12z" />
             <style jsx>{`
                 svg {
-                    fill: ${theme.error};
-                    width: 24px;
+                    fill: inherit;
                     height: 24px;
+                    width: 24px;
+                    vertical-align: middle;
+                    pointer-events: none;
                     margin-right: ${spacers.dp4};
+                    fill: ${theme.error};
```

Comment:
`fill` is duplicated between line 83 and 89, both `inherit` and a colour. Is that intentional?

Reply:
It's the same in other components.

Reply:
Nope that just highlights I shouldn't have tried to make these changes quickly and very late at night... Fixed it now